vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/View.java

```diff
@@ -16,10 +16,24 @@
  * #L%
  */
 
+import java.util.Objects;
+
+/**
+ * View is responsible for centering of the map, rotation or zoom level
```

Comment:
We're not really modifying the map here (centering or rotating), but the viewport for a map.

Also the comment kind of implies an assumption that the view only consists of these 3 things. But in fact there are a lot more. I think we should keep it more general.

Reply:
Done.